content/en/error_tracking/frontend/mobile/reactnative.md

```diff
@@ -5,4 +5,914 @@ code_lang: reactnative
+
+You can also completely override the version by specifying the `version` field. However, make sure you set it correctly, as it has to match the one specified during the upload of your source maps and other mapping files.
+
+For more information about limitations on the version field, see the [Tags documentation][501].
+
+[501]: /getting_started/tagging/#define-tags
+
+{{% /collapse-content %}}
+
+
+
+{{% collapse-content title="Track user interactions" level="h4" %}}
+If user interactions tracking is enabled as in the code example above, the Datadog React Native SDK traverses up the hierarchy of components starting from the component that received a tap, looking for `dd-action-name` property. Once found, it is used as a name for the action reported.
+
+Alternatively, you can use the `accessibilityLabel` element property to give the tap action a name; otherwise, the element type is reported. You can check the sample app for usage examples.
+
+{{% /collapse-content %}}
+
+
+
+{{% collapse-content title="Track background events" level="h4" %}}
+
+<div class="alert alert-info"><p>Track background events may lead to additional sessions, which can impact billing. For questions, <a href="https://docs.datadoghq.com/help/">contact Datadog support.</a></p>
+</div>
+
+You can track events such as crashes and network requests when your application is in the background (for example, when no active view is available).
+
+Add the following snippet during initialization in your Datadog configuration:
+
+```javascript
+configuration.trackBackgroundEvents = true;
+```
+{{% /collapse-content %}}
+
+
+{{% collapse-content title="Send data when device is offline" level="h4" %}}
+
+The React Native SDK ensures availability of data when your user device is offline. In cases of low-network areas, or when the device battery is too low, all events are first stored on the local device in batches. They are sent as soon as the network is available, and the battery is high enough to ensure the React Native SDK does not impact the end user's experience. If the network is not available with your application running in the foreground, or if an upload of data fails, the batch is kept until it can be sent successfully.
+
+This means that even if users open your application while offline, no data is lost.
+
+**Note**: The data on the disk is automatically deleted if it gets too old to ensure the React Native SDK does not use too much disk space.
+
+{{% /collapse-content %}}
+
+
+{{% collapse-content title="Data storage" level="h4" %}}
+##### Android
+
+Before data is uploaded to Datadog, it is stored in cleartext in your application's cache directory. This cache folder is protected by [Android's Application Sandbox][11], meaning that on most devices this data can't be read by other applications. However, if the mobile device is rooted, or someone tampers with the Linux kernel, the stored data might become readable.
```

Comment:
The link on this line is leading to a `404` error, and I'm not sure how it should be updated